lib/main/localizations/app_localizations.dart

```diff
@@ -3280,4 +3280,12 @@ class AppLocalizations {
       name: 'enterSomeSuggestions',
     );
   }
+
+  String marked_single_message_toast(String action) {
+    return Intl.message(
+        'Message has been marked as "$action"',
```

Comment:
As far as design goes, we don't have double quotes here. So delete it.

<img width="761" alt="Screenshot 2023-08-28 at 15 16 45" src="https://github.com/linagora/tmail-flutter/assets/80730648/50d69f69-594d-4e13-af9b-75daf3cf2800">